tests/src/Simple/GC_Filters/GC_Filters.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+
+class Program
+{
```

Comment:
We tend to batch the different test cases that we found to be interesting together in "Simple" tests to make the smoke test runs short. Could you please add this to `tests\src\Simple\Exceptions\Exceptions.cs` instead?